virtcontainers/agent.go

```diff
@@ -197,4 +198,13 @@ type agent interface {
 
 	// statsContainer will tell the agent to get stats from a container related to a Sandbox
 	statsContainer(sandbox *Sandbox, c Container) (*ContainerStats, error)
+
+	// plugRoutes will tell the agent to update route table for an existed Sandbox.
+	plugRoutes(routes []*grpc.Route) error
```

Comment:
Please replace with:
```go
updateRoutes(routes []netlink.Route) error
```
The name will be more appropriate, and the parameter will not be tied to our gRPC protocol.

Reply:
Done. But seems that we will lost grpc.Route.Device info cause it comes from endpoint.Name()

Reply:
I revert this change. Cause input is in the grpc.Route format. Create a new type to decouple it with grpc?